### PR TITLE
Export images

### PR DESCRIPTION
**Pull request by arthurian**
This PR adds a button to `Course Collections > Settings` that allows an admin to download the list of course images as a CSV file. 

Additional notes:
- Updated [gulp](https://gulpjs.com/) to the latest version since I was running into problems with the old version.
- Updated the docker configuration while setting up my local dev environment.
- Deployed and tested in DEV environment.

---
[ATGU-2746](https://jira.huit.harvard.edu/browse/ATGU-2746)

### CODE REVIEW

**Review by joshuagetega — APPROVED** [review]
Looks good to me.

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-FROM python:3.6
+FROM python:3.9-buster
```
Comment (dodget) [inline Dockerfile:1]: Is this the python version running in prod? It may be useful to keep them in sync?

File: app/Dockerfile [hunk 2]
```diff
@@ -1,14 +1,9 @@
-FROM ubuntu:16.04
+FROM ubuntu:20.04
```
Comment (dodget) [inline app/Dockerfile:1]: Again, maybe this should be ubuntu 18 since that is what is running in prod?

**Review by dodget — APPROVED** [review]
Looks good! Just a couple of nitpicks

File: Dockerfile (hunk 1 above)
Comment (arthurian) [reply Dockerfile:1]: Good point, let's keep this in sync. We're still running 3.6 in dev/prod.

File: app/Dockerfile (hunk 2 above)
Comment (arthurian) [reply app/Dockerfile:1]: Agreed - let's stay in sync (I'll revert). It's a bit premature to update until we're actually running 20.04.

**Pull request merged**